src/styles/index.scss

```diff
@@ -1,3 +1,7 @@
-body {
-  margin: 0;
-}
+@import './utils/variables';
+@import './blocks//page';
+@import './blocks//header';
+@import './blocks//nav';
+@import './blocks//main';
```

Comment:
There is a double slash in the import paths for the block styles. Ensure that the paths are correct by using a single slash, like './blocks/page'.